src/features/game/events/landExpansion/removeChicken.test.ts

```diff
@@ -95,11 +95,11 @@ describe("removeChicken", () => {
       coordinates: { x: 2, y: 2 },
       multiplier: 1,
     });
-    expect(gameState.chickens[3]).toEqual({
+    expect(gameState.chickens[3]).toBeUndefined();
```

Comment:
Same as backend. Should change these lookup keys to strings for consistency of type.